test/pluginSpec.js

```diff
@@ -89,14 +89,24 @@ describe('pluginSpec', () => {
         }
       })
 
-      it('should round high-scale amount up to next drop', async function () {
+      it('should not throw if last claim is zero', async function () {
         const encodeSpy = this.sinon.spy(util, 'encodeClaim')
+        this.plugin._lastClaim.amount = '0'
+        this.plugin._channelDetails.balance = '0'
         this.sinon.stub(this.plugin, '_call').resolves(null)
 
         await this.plugin.sendMoney(100)
 
         assert.deepEqual(encodeSpy.getCall(0).args, [ '1', 'abcdef' ])
```

Comment:
Shouldn't it be `'0'`?

Reply:
I'm sending `100`, so it will round up. With this fix, it doesn't encode a claim for the lastClaim if it matches the balance